### PR TITLE
feat: adding no sso config card

### PR DESCRIPTION
**Pull request by kiram15**
Screenshot 2023-02-13 at 12 29 19 PM

[Jira Ticket 
](https://2u-internal.atlassian.net/browse/ENT-6667)
# For all changes

- [x] Ensure adequate tests are in place (or reviewed existing tests cover changes)

# Only if submitting a visual change

- [x] Ensure to attach screenshots
- [ ] Ensure to have UX team confirm screenshots

### CODE REVIEW

File: src/components/settings/SettingsSSOTab/index.jsx [hunk 1]
```diff
@@ -26,6 +29,9 @@ const SettingsSSOTab = ({ enterpriseId, setHasSSOConfig }) => {
         validConfigExists = true;
       }
     });
+    if (existingConfigs?.length < 1) {
```
Comment (marlonkeating) [inline src/components/settings/SettingsSSOTab/index.jsx:32]: I'm a bit curious about the optional chaining construction here.  Do we want this expression to evaluate to false if existingConfigs is undefined or not an array-like object?

Comment (kiram15) [reply src/components/settings/SettingsSSOTab/index.jsx:32]: to be honest this is just logic that is left over and I can add a null check but I feel like we can assume it's an array-like object

**Review by marlonkeating — CHANGES_REQUESTED** [review]
Seeing the 'No SSO' card when I have an incomplete SSO config.

image

File: src/components/settings/SettingsSSOTab/index.jsx [hunk 2]
```diff
@@ -26,6 +29,11 @@ const SettingsSSOTab = ({ enterpriseId, setHasSSOConfig }) => {
         validConfigExists = true;
       }
     });
+    if (existingConfigs?.length < 1 || !existingConfigs) {
```
Comment (marlonkeating) [inline src/components/settings/SettingsSSOTab/index.jsx:32]: nit: Check for falsiness first
```suggestion
    if (!existingConfigs || existingConfigs?.length < 1) {
```

**Review by marlonkeating — CHANGES_REQUESTED** [review]

**Review by marlonkeating — APPROVED** [review]

**Pull request merged**